Frontend/ui-driver/src/Screens/Types.purs

```diff
@@ -886,9 +887,18 @@ type HomeScreenProps =  {
   showAadharPopUp :: Boolean,
   canSendSuggestion :: Boolean,
   showOffer :: Boolean,
-  autoPayBanner :: Boolean
+  autoPayBanner :: Boolean,
+  showAccessbilityPopup :: Boolean
  }
 
+data DisabilityType = BLIND_AND_LOW_VISION | HEAR_IMPAIRMENT | LOCOMOTOR_DISABILITY | OTHER_DISABILITY
```

Comment:
@aryan-13 
0: {tag: ‘BLIND_LOW_VISION’, description: ‘Blind/Low Vision’, id: ‘3b3da2e4-6fa7-b295-8ee9-5991420b28b2’}
1: {tag: ‘HEAR_IMPAIRMENT’, description: ‘Hearing Impairment (Deaf/Mute)’, id: ‘be3560cf-4c98-90b8-2440-badd04c919ac’}
2: {tag: ‘LOCOMOTOR_DISABILITY’, description: ‘Locomotor Disability’, id: ‘e6983b0c-34f0-501d-bd5b-61a0b73b622c’}
3: {tag: ‘OTHER’, description: ‘Other’, id: ‘8a365d73-b81e-6b21-962b-b1397aa687e0’}

Reply:
Can you match this type